Python/RanjanPaudel/task2/migrations/create_table_most_popular_movies.py

```diff
@@ -0,0 +1,23 @@
+import sqlite3
+
+imdb_movies_db = 'sqlite_dbs/imdb_movies.db'
+
+
+def create_table():
+    connection = sqlite3.connect(imdb_movies_db)
+    connection.execute('''CREATE TABLE IF NOT EXISTS most_popular_movies
```

Comment:
You can use context manager here
with con:
con.execute(
Reference:
https://docs.python.org/3/library/sqlite3.html#using-the-connection-as-a-context-manager

Reply:
Please use context manager for other database connection related codes.